app/models/experiment.rb

```diff
@@ -500,6 +500,7 @@ def move_modules(to_move)
       end
 
       my_module.save!
+      experiment.delay.generate_workflow_img
```

Comment:
I'd rather put this outside this loop. And call uniq on experiment_id first. Currently it could be possible if you moved 3 modules to the same experiment, 3 jobs will spawn for the same experiment whereas only one would be enough.